1.js

```diff
@@ -0,0 +1,34 @@
+/*
+1. Написать функцию, которая принимает массив числовых значений, 
+и возвращает минимальное и максимальное значение массива
+*/
+
+"use strict"
+
+let arr = [10, 2, 3, 15, 46, 98];
+
+function maxNum(arr){
+ let max = arr[0];
```

Comment:
please add data type filtering to avoid crashing web app

Reply:
Done